assets/js/common/AvailableSoftwareUpdates/Indicator.jsx

```diff
@@ -24,7 +32,20 @@ function Indicator({ title, critical, tooltip, icon, loading, children }) {
 
   return (
     <Tooltip isEnabled={unknown} content={tooltip} wrap={false}>
-      <div className="flex flex-row items-center border border-gray-200 p-2 rounded-md grow">
+      <div
+        role="button"
+        tabIndex={0}
+        className={classNames(
+          'flex flex-row items-center border border-gray-200 p-2 rounded-md grow',
+          { 'cursor-pointer': !unknown }
+        )}
+        onClick={navigate}
+        onKeyDown={({ code }) => {
+          if (code === 'Enter') {
+            navigate();
+          }
+        }}
+      >
```

Comment:
Great that you considered accessibility! You can't use a `<button />` due to styling limitations, right?